types/single-spa-vue.test-d.ts

```diff
@@ -0,0 +1,26 @@
+import "..";
```

Comment:
What is this for?

Reply:
lol good question. I don't actually know. Its in the [single-spa tsd config](https://github.com/single-spa/single-spa/blob/master/typings/single-spa.test-d.ts#L1) and the official usage just uses a single period `import concat from '.';`.  If I remove it, I get an error saying `Cannot find module 'single-spa-vue' or its corresponding type declarations.ts(2307)`.

Reply:
I guess it's so tsd knows the name of the package